### DISCUSSION TITLE
Packed exe successfully, but running exe file requires pkg file to succeed.

### DISCUSSION THREAD

**Discussion by 0x7c16** [pyinstaller/pyinstaller#6463] [Help] [answered]
I packaged a simple py file, using only the turtle library. The packaging process seems to be fine, but when running the generated exe file, I was reminded "**[7840] Cannot open self C:\Users\xxxx\Desktop\123\dist\jiaozi.exe or archive C:\Users\xxx\Desktop\123\dist\ jiaozi.pkg**". So I had to copy jiaozi.pkg from the "bulid" folder to "dist" in order to run properly,what should I do?

My pyinstaller version is 4.7
python version is 3.10

This is the traceback I get when I use pyinstaller.
![1234](https://user-images.githubusercontent.com/93818523/147378974-f3be0a81-51fa-45bf-9213-72894930c874.png)

**Comment by rokm** [comment, marked as answer]
This looks like an anti-virus program blocked the access to the .exe file before PKG could be appended to it. Are you running an AV program?

**Reply by 0x7c16** [reply]
Yes,  I'm using the the huorong anti-virus program. I took your advice and turned off this anti-virus program. Then ran pyinstaller again and it executed without any problem. Thank you very much for your help! I would like to know how you found the root cause of the problem.
